src/SFA.DAS.ApplyService.InternalApi/Controllers/RoatpAssessorController.cs

```diff
@@ -193,6 +193,107 @@ private static List<int> GetWhatYouWillNeedSectionsForSequence(int sequenceNumbe
         }
 
 
+        [HttpGet("Assessor/Applications/{applicationId}/SectorDetails/{pageId}")]
+        public async Task<SectorDetails> GetSectorDetails(Guid applicationId, string pageId)
```

Comment:
Can this method be broken up a bit?

Reply:
Yes, I'll break it into private methods with clear naming